base/docs/helpdb/Base.jl

```diff
@@ -8908,19 +8856,17 @@ vecnorm
 """
     isvalid(value) -> Bool
 
-Returns `true` if the given value is valid for its type, which currently can be one of
-`Char`, `String`, `UTF16String`, or `UTF32String`.
+Returns `true` if the given value is valid for its type, which currently can be either
+`Char` or `String`.
```

Comment:
This method is deleted for string which breaks IJulia. Is it intentional? (If it is meant to be removed, the doc should be too.)